TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/backfill_rfs.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, config: Dict, target_cluster: Cluster, client_options: Option
         super().__init__(config)
         self.client_options = client_options
         self.target_cluster = target_cluster
-        self.default_scale = self.config["reindex_from_snapshot"].get("scale", 1)
```

Comment:
This pulls `scale` from the services.yaml, let's use that and just set it to the right value in the yaml, which is built by the CDK.

Two places need to be updated:
- For whatever reason, we have the `scale` parameter defined in the Replayer YAML component in the CDK, but not the RFS YAML, so we should add it as a parameter there https://github.com/opensearch-project/opensearch-migrations/blob/main/deployment/cdk/opensearch-service-migration/lib/migration-services-yaml.ts#L64
- And then this is where a `scale` value should be added to that RFS yaml object: https://github.com/opensearch-project/opensearch-migrations/blob/main/deployment/cdk/opensearch-service-migration/lib/service-stacks/reindex-from-snapshot-stack.ts#L189

Reply:
Or, per standup discussion, this could be added as a cdk parameter

Reply:
Thanks for the pointer - I have changed both the default value in the cdk and I'd like to leave this default value to mirror it, any concerns with that approach?